src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -78,11 +78,27 @@
      */
     private Map<String, DetectorData> detectors;
 
+    public Map<String, DetectorData> getDetectors(){
+        return this.detectors;
+    }
+
     /**
      * The time since the last update call.
      */
     private float timeSinceLastUpdate;
 
+    /**
+     * This provides the value for the test
+     * @return a float of timeSinceLastUpdate
```

Comment:
This javadoc is.. weird.

`This provides the value for the test`
Which value? Which test?
`a float of timeSinceLastUpdate`
Makes no sense.

Keep it short and simple.
`Returns the time since last update call.
@returns A float denoting the time since last update call.`
It's fine if there is some redundancy, as long as it's clear.